downloader_library/src/main/res/layout/status_bar_ongoing_event_progress_bar.xml

```diff
@@ -35,7 +36,8 @@
             android:scaleType="centerInside"            
             android:layout_alignParentLeft="true"
             android:layout_alignParentTop="true"            
-            android:src="@android:drawable/stat_sys_download" />
+            android:src="@android:drawable/stat_sys_download"
+            tools:ignore="ContentDescription" />
```

Comment:
Add content description instead of ignoring

Reply:
Done.